cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/index.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { Component } from 'react';
+import {renderGraph} from 'components/OpsDashboard/RunsGraph/graphRenderer';
+import IconSVG from 'components/IconSVG';
+import MyOperationsApi from 'api/operations';
+import {getCurrentNamespace} from 'services/NamespaceStore';
+import moment from 'moment';
+import {parseDashboardData} from 'components/OpsDashboard/RunsGraph/DataParser';
+import ToggleRunsList from 'components/OpsDashboard/RunsGraph/ToggleRunsList';
+
+require('./RunsGraph.scss');
+
+const RUNS_GRAPH_CONTAINER = 'runs-graph-container';
+
+export default class ClassName extends Component {
+  state = {
+    loading: true,
+    data: [],
+    pipelineCount: 0,
+    customAppCount: 0
+  };
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  getData() {
+    let start = moment().subtract(24, 'h').format('x'),
+        duration = 1440;
+
+    start = parseInt(start, 10);
+
+    let params = {
+      start,
+      duration, // 24 hours in minutes
+      namespace: getCurrentNamespace()
+    };
+
+    MyOperationsApi.getDashboard(params)
+      .subscribe((res) => {
```

Comment:
Should we add an error handler to at least set the `loading` state back to false, if the API call returns an error?

Reply:
yeah, this is just for mock API for now.. This method will eventually move to an Action Creator